Atomic/syscontainers.py

```diff
@@ -468,26 +468,28 @@ def _check_oci_configuration_file(self, conf_path, remote=None, include_all=Fals
                         missing_source_paths.append(source)
         return missing_source_paths
 
+
     def _generate_default_oci_configuration(self, destination):
         conf_path = os.path.join(destination, "config.json")
-
-        # If runc is not installed we are not able to generate the default configuration,
-        # write an empty JSON file
-        if not util.runc_available():
+        try:
+            # Try to use $RUNTIME spec to generate a default configuration file.
+            args = [self._get_oci_runtime(), 'spec']
```

Comment:
Hmm, for the run time, could it be bwrap as a runtime here ( e.g: when self.user = true)?  Also, for function `_get_oci_runtime()`, are the only runtimes we have are bwrap and runc?

Reply:
yes it could be bwrap-oci, that is something `_get_oci_runtime()` could return

Reply:
ah, just making sure I get it right, so the idea is that we do not handle error handling /runtime checking in our end anymore. Instead, if the run time does not support spec/exec (i.e bwrap spec), we let bwrap/subprocess call fail and let the subprocess report error?

Reply:
yes correct.  Since we can't know in advance whether the runtime supports that operation or not.